### ISSUE TITLE
Remove friction when adding a new film

### ISSUE THREAD

**Issue by C0derLint** [greybax/awesome-IT-films#67]
With #63 we have automatic ratings, but this includes way more steps required to go through when you add a new movie. 

My 2 cents:
 - Generate `storage.js` from `_prebuilt.md` - The user is going to add the IMDb link when adding a movie. We could simply parse the file and add this into `storage.js`
 - Make `Contributing.md` more understandable. I had to hunt for `_prebuilt.md`
 - `Then run npm run build-movies` -> This is assuming that whoever is adding the movie knows how to use npm to install dependencies. Correct me if I'm wrong, but the user needs a local api-key?
 - Use something like Travis CI to build instead. I'm thinking of a pipeline like this: `User modifies prebuilt.md and sends PR` > `Accept PR` > CI automatically generates `storage.js` from `_prebuilt.md`. Then it runs npm build-movies and deploys it back to branch.

**Comment by greybax** [comment]
> Generate storage.js from _prebuilt.md - The user is going to add the IMDb link when adding a movie. We could simply parse the file and add this into storage.js

yes, this could be a better solution than we have now. Don't forget PRs are always welcome! ;)

> Make Contributing.md more understandable. I had to hunt for _prebuilt.md

Will think about it and as I said before PRs are always welcome

> Then run npm run build-movies -> This is assuming that whoever is adding the movie knows how to use npm to install dependencies. Correct me if I'm wrong, but the user needs a local api-key?

If you want to run npm locally, so yes you need api key. 

> Use something like Travis CI to build instead. I'm thinking of a pipeline like this: User modifies prebuilt.md and sends PR > Accept PR > CI automatically generates storage.js from _prebuilt.md. Then it runs npm build-movies and deploys it back to branch.

Actually it alsmost how it works now. But we use Codeship instead of Travis.
